diff --git a/main/configure.ac b/main/configure.ac
index 2ee270eba6..6370bf349b 100644
--- a/main/configure.ac
+++ b/main/configure.ac
@@ -246,9 +246,6 @@ AM_CONDITIONAL(ENABLE_WINDOWSPLATFORM, [test x$enable_windowsplatform = xyes])
 
 if test x$enable_gnomeplatform = xyes && test x$enable_git = xyes; then
   AC_PATH_PROG(CMAKE, cmake, no)
-  if test "x$CMAKE" = "xno"; then
-          AC_MSG_ERROR([You need to install cmake for Git support to be built])
-  fi
 
   PKG_CHECK_MODULES(LIBSSH2, libssh2)
   AC_SUBST(LIBSSH2_LIBS)
